src/models/helpmodel.cpp

```diff
@@ -43,10 +43,10 @@ void maybeInitialize() {
   s_helpEntries.append(
       HelpEntry("help.helpCenter2", true, false, MozillaVPN::LinkHelpSupport));
 
-  //% "Contact us"
-  logger.debug() << "Adding:" << qtTrId("help.contactUs");
-  s_helpEntries.append(
-      HelpEntry("help.contactUs", false, false, MozillaVPN::LinkContact));
+  logger.debug() << "Adding:"
+                 << qtTrId("vpn.inAppSupportWorkflow.supportNavLinkText");
+  s_helpEntries.append(HelpEntry("vpn.inAppSupportWorkflow.supportNavLinkText",
```

Comment:
I don't like this because it exposes the string ID generated from the `strings.yaml`. I wonder if we can do something different:
- we can add a `const char* id(String)` method in ` l18nstrings.h`.
- here we can use it in this way: `s_helpEntries.append(HelpEntry(L18nStrings::instance()->id(L18nStrings::InAppSupportWorkflowSupportNavLinkText)))`

It's a bit more work because you need to change `scripts/utils/generate_strings.py`: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/scripts/utils/generate_strings.py#L327 adding `const char* id(String) const;`

and also: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/translations/l18nstrings.cpp#L39
Adding something like:

```
const char* L18nString::id(L18nString::String string) const {
  Q_ASSERT(string < __Last);
   return _ids[string];
}
```

Actually! If you add this method, you can also simplify the `t()` one:

```
                                                     
QString L18nStrings::t(L18nStrings::String string) const {                                                       
  QString id = id(string);                                                                                     
  if (id.isEmpty()) return "";                                                                                   
  return qtTrId(id);                                                                                   
}                                                              
```